src/Header.js

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components'
+
+function Header() {
+  return (
+    <Wrapper>
+      <Image src="/img/Schriftzug.jpg" alt="" />
+    </Wrapper>
+  )
+}
+
+const Wrapper = styled.section`
```

Comment:
Instead of using "section" I would choose "header", because this is more semantic. 🤔